Issue with check-export in SCE

#### Description of problem:
While testing check-export in sce, we have an issue. value-id and export-name are inverted.

#### SCAP Security Guide Version:
latest (future 0.1.58)

#### Operating System Version:
Fedora

#### Steps to Reproduce:

1. add a check-export annotation to file linux_os/guide/system/accounts/accounts-session/accounts_users_own_home_directories/sce/ubuntu2004.sh 
  
"# check-export = TESTENVVAR=var_accounts_tmout"

2. build with make -j4 ubuntu2004
3. build fail

[build]$ make -j4 ubuntu2004
[  0%] [ubuntu2004-content] generating sce/metadata.json
[  0%] Built target ubuntu2004-tables
[  0%] Built target generate-internal-ubuntu2004-sce-metadata.json
[  0%] [ubuntu2004-content] compiling profiles
[  0%] [ubuntu2004-content] generating shorthand.xml
[  0%] Built target generate-internal-ubuntu2004-shorthand.xml
[  0%] [ubuntu2004-content] generating xccdf-unlinked-resolved.xml
[  0%] Generating ../tables/tables-ubuntu2004-all.html
[  0%] [ubuntu2004-content] generating templated content
File '/home/mgjadoul/GIT/content-latest/build/ubuntu2004/xccdf-unlinked-resolved.xml' line 4813: Element '{http://checklists.nist.gov/xccdf/1.1}check-export': No match found for key-sequence ['TESTENVVAR'] of keyref '{http://checklists.nist.gov/xccdf/1.1}valueIdKeyRef'.
OpenSCAP Error: Invalid XCCDF Checklist (1.1) content in /home/mgjadoul/GIT/content-latest/build/ubuntu2004/xccdf-unlinked-resolved.xml. [/builddir/build/BUILD/openscap-1.3.5/src/source/oscap_source.c:353]
make[3]: *** [ubuntu2004/CMakeFiles/generate-internal-ubuntu2004-xccdf-unlinked-resolved.xml.dir/build.make:77 : ubuntu2004/xccdf-unlinked-resolved.xml] Erreur 1
make[3]: *** Suppression du fichier « ubuntu2004/xccdf-unlinked-resolved.xml »
make[2]: *** [CMakeFiles/Makefile2:40663 : ubuntu2004/CMakeFiles/generate-internal-ubuntu2004-xccdf-unlinked-resolved.xml.dir/all] Erreur 2
make[2]: *** Attente des tâches non terminées....
[  0%] Built target generate-ssg-tables-ubuntu2004-all
[  0%] Built target generate-internal-templated-content-ubuntu2004
make[1]: *** [CMakeFiles/Makefile2:39737 : ubuntu2004/CMakeFiles/ubuntu2004.dir/rule] Erreur 2
make: *** [Makefile:14162 : ubuntu2004] Erreur 2


#### Actual Results:
Build fail.
By analyzing shorthand generated, we can see that in XML file the value-id and export-name are inverted.


#### Expected Results:
As per documentation: 
check-export: a comma-separated list of env_variable=xccdf_variable pairs to export via XCCDF <check-export /> elements.


#### Additional Information/Debugging Steps:

This is confirmed by inverting the element in the annotation: the build works in this case:

"# check-export = var_accounts_tmout=TESTENVVAR"

--- a/ssg/build_yaml.py
+++ b/ssg/build_yaml.py
@@ -1478,7 +1478,7 @@ class Rule(object):
                 if isinstance(self.sce_metadata['check-export'], str):
                     self.sce_metadata['check-export'] = [self.sce_metadata['check-export']]
                 for entry in self.sce_metadata['check-export']:
-                    value, export = entry.split('=')
+                    export, value = entry.split('=')
                     check_export = ET.SubElement(check, 'check-export')
                     check_export.set('value-id', value)
                     check_export.set('export-name', export)

